docs/config.rst

```diff
@@ -174,9 +286,51 @@ Java Kafka clients.
   * Default: 1000
   * Importance: low
 
+``simpleconsumer.max.poll.records``
+  Maximum number of records that can be fetched by a single consumer poll request. Since desired number of fetched records can't be defined for every poll request the consumer can poll more records thannecessary (extra records are stored in cache.). The value of this property defines maximum number of excess records fetched by a single user request. The grater value means grater network overhead. It may be reasonable if user retrieves records from specified partition increasing offset sequentially. The smaller value suits better when the user consumes records in a random fashion.Use 0 for no max value allowed by kafka consumer 2147483647
+
+  * Type: string
+  * Default: "100"
+  * Importance: low
+
+``simpleconsumer.max.poll.time``
```

Comment:
I agree that it is verbose to use `simpleconsumer.max.poll.records` as there is already `simpleconsumer.cache.max.records`.